src/i18n/langs/cs.json

```diff
@@ -1,301 +1,316 @@
 {
   "translation": {
-    "navigation": {
-      "home": "Domů",
-      "dark_mode": "Tmavý režim",
-      "settings": "Nastavení",
-      "back": "Zpět",
-      "logout": "Odhlásit se",
-      "apps": "Aplikace",
-      "display_sats": "Zobrazit SATS"
+    "appInfo": {
```

Comment:
Not sure if it's necessary to sort these alphabetically, but fine for me

Reply:
Yes it was not on purpose. The tool I work with localy did that. I used Babel Edit for that:
https://www.codeandweb.com/babeledit